src/lib/button/button.scss

```diff
@@ -79,6 +80,10 @@
   border-radius: inherit;
   pointer-events: none;
   opacity: 0;
+
+  @include md-high-contrast {
+    background-color: rgba(white, 0.5);
```

Comment:
Could this be an issue with dark themes (where the text is white)?

(same for other components where you use a white background)